src/components/Form/UploadDocumentForm.jsx

```diff
@@ -125,10 +138,18 @@ const UploadDocumentForm = () => {
       defaultMessage="To be uploaded..."
     >
       <form onSubmit={handleFormSubmission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc">Select document type to upload: </label>
-          <DocumentSelection htmlId="upload-doc" />
-        </div>
+        <Box sx={{ minWidth: 120 }}>
+          <FormControl fullWidth autoComplete="off">
```

Comment:
The FormControl in Line 142 are not implemented correctly. `<FormControl>` is suppose to replace `<form>` in Line 140 not the `<label>` element (we have `<InputLabel>` for that).

See how example in FetchDocumenForm.jsx where the `<form>` element is completely replaced by `<FormControl>` and where `<input>` is replaced with `<InputLabel>`